Firestore/Protos/README.md

```diff
@@ -1,5 +1,15 @@
 ## Usage
 
+First, build protobuf and nanopb
+```
+cd firebase-ios-sdk
```

Comment:
I tried that, but unfortunately the pod for nanopb doesn't build the (python) protoc plugin, so you can't use it as is to generate the nanopb protos.

So if we want ios-only developers to (a) not need cmake and (b) also regenerate the protos, then we're going to have to jump through a few more hoops. (i.e. we're going to have to build nanopb ourselves rather than relying on the pod, or create a new pod, or ...?)

Reply:
Ew. I see what you mean.

Once we go down the route of building nanopb for ourselves we're probably better off just doing it via CMake :-(.

However, that does mean that we should probably make sure that the version we build via CMake is the same version we get via CocoaPods. I think that means that unfortunately we'll need to `./configure` and `make` rather than relying on the nanopb CMakeLists.txt